### PR TITLE
fixes #4186: foreman-rake db:seed fails in 1.4rc2

### PR DESCRIPTION
**Pull request by shoekstra**
fixes "Unable to create medium: Path has already been taken" in #4186.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -115,7 +115,7 @@ def audit_modified?(type, name)
     { :name => "OpenSUSE mirror", :os_family => "Suse", :path => "http://download.opensuse.org/distribution/$major.$minor/repo/oss", :operatingsystems => os_suse },
     { :name => "Ubuntu mirror", :os_family => "Debian", :path => "http://archive.ubuntu.com/ubuntu/" }
   ].each do |input|
-    next if Medium.find_by_name(input[:name])
+    next if Medium.where(['name = ? OR path = ?',input[:name], input[:path]]).any?
```
Comment (lzap) [inline db/seeds.rb]: Missing space after the first comma?

Works for me.

**Comment by lzap** [comment]
Just to be sure: [test]

**Comment by domcleal** [comment]
[test]

**Comment by domcleal** [comment]
Thanks for the fix @shoekstra, merged as 202770a for Foreman 1.4.0!

**Pull request closed without merging**